Resources/Prototypes/Entities/Mobs/NPCs/Rslimes.yml

```diff
@@ -234,56 +234,56 @@
   - type: VentCrittersRule
     entries:
     - id: ReagentSlime
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeBeer
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimePax
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeNocturine
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeTHC
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeBicaridine
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeToxin
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeNapalm
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeOmnizine
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeMuteToxin
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeNorepinephricAcid
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeEphedrine
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeRobustHarvest
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeIchor
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeBleach
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeSoap
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeSpacelube
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeBuzzachloricbees
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeWehjuice
-      prob: 0.01
+      prob: 0.002
     - id: ReagentCognizine
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeNecrosol
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeSpaceDrugs
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeUnstableMutagen
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeLead
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimechlorinetriflouride
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimePotassium
-      prob: 0.01
+      prob: 0.002
     - id: ReagentSlimeLotophagoiOil
-      prob: 0.01
+      prob: 0.002
```

Comment:
Isn't this no different than before? or is this general "will this spawn" probability?

Reply:
Oh the way this works is absolutely stupid and hard to explain. But basically **every possible spawn location** goes down the entire full list of potential spawnables. But this will quite literally mean it spawns 1/5th as many slimes.

Reply:
I was not aware of this. good to know